src/ru/skypro/EmployeeService.java

```diff
@@ -0,0 +1,229 @@
+package ru.skypro;
+
+public class EmployeeService {
+
+
+    public static boolean isNullEmployee(Employee employee) {
+        if (employee != null) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public static boolean isNullEmployeeList(Employee[] employees) {
+        if (employees[0] == null) {
+            return false;
+        } else {
+            return true;
+        }
+    }
+
+    public static int getCountInList(Employee[] employees) {
+        int count = 0;
+        for (Employee employee : employees) {
+            if (employee != null) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static boolean isDepartmentExists(Employee[] employees, int department) {
+        for (Employee employee : employees) {
+            if (employee.getDepartment() == department) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static void add(Employee[] employees, Employee employee) {
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] == null) {
+                employees[i] = employee;
+                break;
+            }
+        }
+    }
+
+    public static void printEmployees(Employee[] employees) {
+
+        for (Employee employee : employees) {
+            if (isNullEmployee(employee)) {
+                System.out.println(employee.toString());
```

Comment:
```suggestion
                System.out.println(employee;
```
toString будет вызван автоматически